setup.py

```diff
@@ -1,5 +1,19 @@
+import pip
 import setuptools
 
+# parse "./requirements.txt" using pip's `parse_requirements`
+pip_major_version = list(map(int, pip.__version__.split('.')))[0]
+if pip_major_version >= 10:
+    from pip._internal.req import parse_requirements
+else:
+    from pip.req import parse_requirements
+
+install_reqs = parse_requirements('requirements.txt', session=False)
+if pip_major_version >= 20:
+    requirements = [str(ir.requirement) for ir in install_reqs]
```

Comment:
I get the following error when I try to run `python setup.py install`. I have pip 20.0.2.

```
Traceback (most recent call last):
  File "setup.py", line 13, in <module>
    requirements = [str(ir.requirement) for ir in install_reqs]
  File "setup.py", line 13, in <listcomp>
    requirements = [str(ir.requirement) for ir in install_reqs]
AttributeError: 'InstallRequirement' object has no attribute 'requirement'
```

Reply:
This seems to be fixed now.